cmd/kubeadm/app/util/endpoint.go

```diff
@@ -34,22 +34,10 @@ import (
 // - if the controlPlaneEndpoint is defined but without a port number, use the controlPlaneEndpoint + localEndpoint.BindPort is used.
```

Comment:
these docs are now out of date w/ respect to how this method works

Reply:
this can be fixed in the master branch.

Reply:
@jdef actually...why do you think the docs of the function are no longer valid?

all comments seem guaranteed to me by the unit tests that we have:
```
// GetControlPlaneEndpoint returns a properly formatted endpoint for the control plane built according following rules:
// - If the controlPlaneEndpoint is defined, use it.
// - if the controlPlaneEndpoint is defined but without a port number, use the controlPlaneEndpoint + localEndpoint.BindPort is used.
// - Otherwise, in case the controlPlaneEndpoint is not defined, use the localEndpoint.AdvertiseAddress + the localEndpoint.BindPort.
```

https://github.com/neolit123/kubernetes/blob/d9441906c4155173ce1a75421d8fcd1d2f79c471/cmd/kubeadm/app/util/endpoint_test.go#L31-L196